lib/cog_api/http/relay_groups.ex

```diff
@@ -77,6 +66,26 @@ defmodule CogApi.HTTP.RelayGroups do
   def update_assignments(action, relay_group_id, bundle_id, endpoint) when is_binary(bundle_id),
     do: update_assignments(action, relay_group_id, [bundle_id], endpoint)
 
+  defp get_relay_ids(relay_names, endpoint) when is_list(relay_names) do
```

Comment:
It'd be nice to abstract this "turn names into IDs" behavior into its own function, since it's the same for relays and bundles, particularly if this same pattern is also being used elsewhere, which I suspect it probably is.